bluemira/equilibria/fem_fixed_boundary/utilities.py

```diff
@@ -350,11 +351,10 @@ def get_flux_surfaces_from_mesh(
             else:
                 index.append(i)
 
-    n = len(index)
-    for xi in range(n):
-        x_1d = np.delete(x_1d, index[xi])
-
-    return x_1d, flux_surfaces
+    mask = np.ones_like(x_1d, dtype=bool)
+    mask_index = index + np.arange(len(index))
+    mask[mask_index[mask_index < len(x_1d)]] = False
```

Comment:
Right but, we're confident this was a bug, so why leave it in? Can we at least put a comment or TODO relating to this which needs to be fixed.

Reply:
fair enough I'll take it out (and open an issue)

Reply:
#2574